Build failure: unnamed-sdvx-clone

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Try to build unnamed-sdvx-clone

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/310012410

### Relevant log output

```console
error: Cannot build '/nix/store/s7mkbh3jpdijj2i6wxig337svlvrij1j-unnamed-sdvx-clone-0.6.0.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/vqjlwb75nvhi44dfqrpxjfban85hsv1s-unnamed-sdvx-clone-0.6.0
       Last 25 log lines:
       > -- Detecting CXX compile features
       > -- Detecting CXX compile features - done
       > -- Found Freetype: /nix/store/5zszfvll8z3j72n42vm59rb70lfrlqmk-freetype-2.13.3/lib/libfreetype.so (found version "2.13.3")
       > -- Found ZLIB: /nix/store/6qi8skh85ci2k9gvl27nnh3kiy32qnsz-zlib-1.3.1/lib/libz.so (found version "1.3.1")
       > -- Found SDL2: /nix/store/jhn5x6c698ri1lp3n00xgnwvnrvhkpig-sdl2-compat-2.32.56-dev/include
       > -- Found PNG: /nix/store/zz7vx2a97vyi8r1ap6znzdkq5mr9bvrw-libpng-apng-1.6.50/lib/libpng.so (found version "1.6.50")
       > -- Found JPEG: /nix/store/8pm15cppsc99va4csmp5cywk1mwsn7zl-libjpeg-turbo-3.1.2/lib/libjpeg.so (found version "62")
       > -- Found Vorbis: /nix/store/3sir2khg7r7n6a6mcqgpzfbbh89i8mfk-libogg-1.3.6-dev/include
       > -- Found OGG: optimized;/nix/store/04ahing7076i610yw0wyn536y2q1hfvb-libogg-1.3.6/lib/libogg.so;debug;/nix/store/04ahing7076i610yw0wyn536y2q1hfvb-libogg-1.3.6/lib/libogg.so
       > -- Found LibArchive: /nix/store/aakri4kzhxb04hij6fz99a2r8d8sbbxa-libarchive-3.8.1-lib/lib/libarchive.so (found version "3.8.1")
       > -- Performing Test Iconv_IS_BUILT_IN
       > -- Performing Test Iconv_IS_BUILT_IN - Success
       > -- Found Iconv: built in to C library
       > Enabling crash dumps
       > CMake Error at third_party/lua-5.3.5/CMakeLists.txt:2 (cmake_minimum_required):
       >   Compatibility with CMake < 3.5 has been removed from CMake.
       >
       >   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
       >   to tell CMake that the project requires at least <min> but has been updated
       >   to work with policies introduced by <max> or earlier.
       >
       >   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
       >
       > 
       > -- Configuring incomplete, errors occurred!
```

### Additional context

This is due to lua 5.3.5 in the third party folder having ```cmake_minimum_required(VERSION 3.4)```.
Related to #445447 .

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.49, NixOS, 25.11 (Xantusia), 25.11.20250928.e9f00bd`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - nixpkgs: `/nix/store/jc3mfa7ybzh32bkvdj5xiib1vkx8jy6x-source`

### Notify maintainers

@Sakooooo 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/un/unnamed-sdvx-clone/package.nix
+++ b/pkgs/by-name/un/unnamed-sdvx-clone/package.nix
@@ -25,14 +25,14 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "unnamed-sdvx-clone";
-  version = "0.6.0";
+  version = "0.6.0-unstable-2025-07-31";
 
   src = fetchFromGitHub {
     owner = "Drewol";
     repo = "unnamed-sdvx-clone";
-    tag = "v${finalAttrs.version}";
+    rev = "2472fcc5c8b6b8bc7323dc61ea80b3b1899dd45f";
     fetchSubmodules = true;
-    hash = "sha256-wuf7xZztoxzNQJzlJOfH/Dc25/717NevBx7E0RDybho=";
+    hash = "sha256-Z0eU3xNzVmv2PwUvS2HoRe2TQzQ8seL0R/YAYCQWR28=";
   };
 
   nativeBuildInputs = [

